.circleci/config.yml

```diff
@@ -2,12 +2,18 @@
 #
 # Check https://circleci.com/docs/2.0/language-java/ for more details
 #
-version: 2
-jobs:
-  build:
+version: 2.1
+
+executors:
+  openjdk8:
     docker:
       - image: circleci/openjdk:8-jdk
-
+  integration:
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
Can you add a comment on why we need a machine instead of a docker image? I guess it's because of testcontainers?

Reply:
Also FWIW, testcontainers work well on github actions :) We should migrate this build sooner than latter

Reply:
updated integration-test job w/ comment.  I agree and won't plan on any additional major CI additions without having moved to actions beforehand.